### ISSUE TITLE
[BUG] Playwright is not being able to click a link with user-select: none;

### ISSUE THREAD

**Issue by kblok** [microsoft/playwright#10920]
**Context:**
- Playwright Version: 1.17.0
- Operating System: Mac
- Node.js version: 14.17
- Browser: Chromium 96 Channel Stable (it works on the bundled chromium)

I'm not able to click a link with `user-select: none;`.
I tried to isolate the issue. But It seems it's not as simple as adding `user-select: none;` to any link:
This piece of code works as expected:

```
const browser = await playwright.chromium.launch({headless: false, channel: 'chrome'});
const page = await browser.newPage();

await page.setContent(`
  <a class="clickMe" style="user-select: none;" href="javascript:markClick();">Click Me</a>
  <span id="clicked">No</span>
  <script language="javascript">
    function markClick() {
      document.getElementById('clicked').innerText = 'Yes';
    }
  </script>
`);

await page.click('.clickMe');
```

But I have a real scenario which is failing:

```
(async () => {
  const browser = await playwright.chromium.launch({headless: false, channel: 'chrome'});
  const page = await browser.newPage();

  await page.goto('https://bit.ly/3DX1cLo');
  //await page.$eval('xpath=/html/body/footer/nav[2]/ul/li[3]/a', el => el.setAttribute('style', ''));
  await page.click('xpath=/html/body/footer/nav[2]/ul/li[3]/a');
  await page.waitForTimeout(10000);
  await browser.close();
})();
```
That link is supposed to open the panel at the bottom of the page. If you uncomment the `$eval` that will remove the `user-select: none,` the script will work.

**Comment by mxschmitt** [comment]
Could you try version `1.17.1`? I tried to repro but was not able to and we fixed a click related issue in 1.17.1.

**Comment by kblok** [comment]
1.7.1 fixed it. Thank you @mxschmitt!
